cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -151,6 +151,16 @@ write_files:
         done;
         export HOME="${!HOME_BAK}"
       }
+
+      function measure_startup_time
+      {
+        METRIC_NAMESPACE="ParallelCluster"
+        METRIC_NAME="StartupTime"
+        INSTANCE_ID=$(cat /etc/parallelcluster/slurm_plugin/slurm_node_spec.json | jq -r .instance_id)
+        INSTANCE_TYPE=$(cat /etc/parallelcluster/slurm_plugin/slurm_node_spec.json | jq -r .'compute."instance-type"')
+        aws cloudwatch put-metric-data --namespace $METRIC_NAMESPACE --metric-name $METRIC_NAME --value $(($2-$1)) --dimensions InstanceID=$INSTANCE_ID,InstanceType=$INSTANCE_TYPE --region "${AWS::Region}"
```

Comment:
Will adding `cluster_name` as one of the dimension make it easier to graph the metrics on cw dashboard if there are multiple clusters in the account?